### ISSUE TITLE
`cannot import name 'etree' from 'markdown.util'`

### ISSUE THREAD

**Issue by dragoncoder047** [getpelican/pelican#3019]
- [X] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [X] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

* **OS version and name**: Whatever `ubuntu-latest` is on Github Actions
* **Python version**: 3.10.4, 3.10.5 (same error both wars)
* **Pelican version**: 4.8.0
* **Link to theme**: https://github.com/dragoncoder047/pelicantheme
* **Links to plugins**: https://github.com/dragoncoder047/blog/blob/main/build.py#L160-L170
* **Link to your site**: https://dragoncoder047.github.io/blog (but there's nothing there right now!)
* **Link to your source**: https://github.com/dragoncoder047/blog/tree/main/aaaa/
* Github Actions run: https://github.com/dragoncoder047/blog/runs/7394422856?check_suite_focus=true#step:7:8

## The Bug

EVery single file error's out with this error:

```
           ERROR    Could not process <FILE>                           log.py:96
                    cannot import name 'etree' from 'markdown.util'             
                    (/opt/hostedtoolcache/Python/3.10.4/x64/lib/python          
                    3.10/site-packages/markdown/util.py)
```

and it winds up with `Done: Processed 0 articles, 0 drafts, 0 hidden articles, 0 pages, 0 hidden pages and 0 draft pages in X.XX seconds.`

What the heck is going on???

**Comment by dragoncoder047** [comment]
Built at home using Pelican 4.7.2; it worked fine. 

Commit: https://github.com/dragoncoder047/blog/commit/5d9f99e4f50493e960b7c421bb82bccea3b265d1

**Comment by justinmayer** [comment]
Sorry to hear you had some trouble. As odd as it may seem, the problem has nothing to do with Pelican. As noted in the error itself, it originates from the Python-Markdown library that Pelican uses to process Markdown content.

If you were to look through recent issues in the Python-Markdown repository, you would see an issue that enumerates [many Markdown extensions that no longer work with the just-released Python-Markdown 3.4 version](https://github.com/Python-Markdown/markdown/issues/1277). They don't work because of long-deprecated APIs that have finally been removed entirely, and the affected extensions haven't been updated appropriately. The error in this issue is clearly related to [those removed objects](https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/release-3.4.md#previously-deprecated-objects-have-been-removed), so some Markdown extension you are using is in need of updating.

You could downgrade via `pip install markdown==3.3.7`. Or you could help the affected Markdown extension's maintainer to update their extension(s). Or you could remove the affected extension(s). I think that covers the available options for remediation.

Hope this helps!

**Comment by dragoncoder047** [comment]
Thank you! that worked.
